src/likelihoods/negativebinomial.jl

```diff
@@ -1,36 +1,170 @@
+abstract type NBParam end
+default_invlink(::NBParam) = error("Not implemented")
```

Comment:
Is there something worthwhile we can add in the error message? If not, maybe just remove it ("Not implemented" seems to me less helpful than the Julia default MethodError which at least suggests what other methods might be available...)
```suggestion
```